src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,103 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    //WebElements
+    @FindBy(css = "img.avatar")
+    WebElement userAvatarIcon;
+    @FindBy(css = "i[class='fa fa-plus-circle create']")
+    WebElement plusIcon;
+    @FindBy(css = "li[data-testid='playlist-context-menu-create-simple']")
+    WebElement newPlaylistButton;
+    @FindBy(css = "input[name='name']")
+    WebElement playlistInputField;
+    @FindBy(css = "[class='del btn-delete-playlist']")
+    WebElement deleteButton;
+    @FindBy(css = "div.success.show")
+    WebElement successMsg;
+    @FindBy(xpath = "//li[contains(text(),'Edit')]")
+    WebElement editPlaylistButton;
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example
```
    @FindBy(css = "img.avatar")
    private WebElement userAvatarIcon;
    @FindBy(css = "i[class='fa fa-plus-circle create']")
    private WebElement plusIcon;
    @FindBy(css = "li[data-testid='playlist-context-menu-create-simple']")
    private WebElement newPlaylistButton;
    @FindBy(css = "input[name='name']")
   private WebElement playlistInputField;
    @FindBy(css = "[class='del btn-delete-playlist']")
    WebElement deleteButton;
    @FindBy(css = "div.success.show")
    private WebElement successMsg;
    @FindBy(xpath = "//li[contains(text(),'Edit')]")
    private WebElement editPlaylistButton;
```